internal/envoygateway/scheme.go

```diff
@@ -11,7 +11,8 @@ import (
 	gwapiv1a2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
 	gwapiv1b1 "sigs.k8s.io/gateway-api/apis/v1beta1"
 
-	"github.com/envoyproxy/gateway/api/config/v1alpha1"
+	egcfgv1a1 "github.com/envoyproxy/gateway/api/config/v1alpha1"
```

Comment:
can we combine these two into one ?

Reply:
Shouldn't we keep them separate to simplify moving the Gateway API extensions, e.g. AuthenticationFilter, out of EG into a separate repo in the future?

Reply:
trying to highlight the fact that current dir structure is confusing - either we have `api/extensions/v1alpha1` or just `api/v1alpha1` .

Reply:
Agreed. Per your request, I removed the `security` group from the Request Authentication design. I'll add this to the community meeting agenda to get input from other maintainers.

Reply:
xref https://github.com/envoyproxy/gateway/issues/809 to make a decision on API type grouping. IMO a separate PR should be submitted to resolve this issue.